### PR TITLE
[16.0][MIG] account_invoice_import_ubl

### PR DESCRIPTION
**Pull request by alexis-via**
Requires https://github.com/OCA/edi/pull/725
I'll publish account_invoice_import to v16 later on.

### CODE REVIEW

**Comment by acsonefho** [comment]
@alexis-via thank you for this PR :)
Do you already know when you'll open a PR for `account_invoice_import` ?
Thanks!

File: account_invoice_import_ubl/wizard/account_invoice_import.py [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+            "currency": {"iso": currency_iso_xpath[0].text},
+            "amount_total": amount_total,
+            "amount_untaxed": amount_untaxed,
+            "iban": iban_xpath and iban_xpath[0].text or False,
+            "bic": bic_xpath and bic_xpath[0].text or False,
+            "lines": res_lines,
+            "attachments": attachments,
+        }
+
+        self.get_currency_helper(res)
```
Comment (acsonefho) [inline account_invoice_import_ubl/wizard/account_invoice_import.py:254]: Where does it come from ?

**Review by acsonefho — CHANGES_REQUESTED** [review]

**Comment by tarteo** [comment]
This functionality is already in core odoo (account_edi_ubl_cii)

**Comment by alexis-via** [comment]
For some strange reasons, I can't reopen this PR. So I'll create a new one.

**Pull request closed without merging**